src/Select.jsx

```diff
@@ -1,108 +1,134 @@
 import React, { useState, useEffect } from 'react';
 import Autocomplete from '@material-ui/lab/Autocomplete';
-import { TextField } from '@material-ui/core';
+import TextField from '@material-ui/core/TextField';
```

Comment:
Is there a reason why you decided to import components with:

```jsx
import TextField from '@material-ui/core/TextField';
import CircularProgress from '@material-ui/core/CircularProgress';
```

instead of:

```jsx
import { TextField, CircularProgress } from '@material-ui/core';
```

Reply:
https://material-ui.com/guides/minimizing-bundle-size/#option-1